hal/transport/RFM69/driver/new/RFM69_new.cpp

```diff
@@ -565,6 +644,69 @@ LOCAL bool RFM69_standBy(void)
 }
 
 
+LOCAL void RFM69_sendACK_fast(const uint8_t recipient, const rfm69_sequenceNumber_t sequenceNumber,
```

Comment:
When should RFM69_sendACK_fast be used and when should RFM69_sendACK be used? Is there a reason both need to exist?

Reply:
I tried to not touch current rfm69 new driver workflow. So if MY_RX_MESSAGE_BUFFER_FEATURE is not defined old logic works. If MY_RX_MESSAGE_BUFFER_FEATURE is defined workflow is differ. In this case incoming messages are processed by interrupt. Ack messages are sent inside interrupt too (RFM69_sendACK_fast) so there is not possible to freeze the thread or use time functions.

Reply:
👍 Thanks for clarifying.